osf/utils/fields.py

```diff
@@ -1,4 +1,5 @@
 import jwe
+import json
 from django.db import models
 from django.contrib.postgres.fields.jsonb import JSONField
```

Comment:
And there are two more usage of `django.contrib.postgres.fields.jsonb` that are missed. One is here and the other is in in https://github.com/CenterForOpenScience/osf.io/blob/develop/api/files/annotations.py#L3

```
from django.contrib.postgres.fields.jsonb import KeyTextTransform
```

The JSON field stuff will be removed in 4.0 (only deprecated starting 3.1). However, if we are upgrading this one now, why not upgrade all places? @Johnetordoff ? Any blockers?